stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/CloseDelimitedRecordChannel.java

```diff
@@ -34,11 +34,13 @@
  */
 @BallerinaFunction(
         packageName = "ballerina.io",
-        functionName = "closeTextRecordChannel",
-        receiver = @Receiver(type = TypeKind.STRUCT, structType = "TextRecordChannel", structPackage = "ballerina.io"),
+        functionName = "closeDelimitedRecordChannel",
+        receiver = @Receiver(type = TypeKind.STRUCT,
+                structType = "DelimitedRecordChannel",
+                structPackage = "ballerina.io"),
         isPublic = true
 )
-public class CloseTextRecordChannel extends AbstractNativeFunction {
+public class CloseDelimitedRecordChannel extends AbstractNativeFunction {
 
     /**
      * The index of the TextRecordChannel in ballerina.io#closeTextRecordChannel().
```

Comment:
We need to change the comment to reflect the new method name change.

Reply:
c65b5580d110f274fb2a630c136996a31c17d89e